internal/services/servicebus/servicebus_queue_data_source_test.go

```diff
@@ -83,9 +83,8 @@ func (ServiceBusQueueDataSource) withNamespaceName(data acceptance.TestData) str
 %s
```

Comment:
is this function needed anymore, now that namespace_name is deprecated?

Reply:
You can make it Required but you have to set it as Optional in the FivePointOhBeta block so it's not a breaking change until 5.0 hits.

Reply:
It is still necessary just so we can confirm that this functionality still works